_config/extensions.yml

```diff
@@ -1,7 +1,7 @@
 ---
 Name: cwpcoreextensions
 ---
-# @todo Disable the feature. LoginAttempt seems to be broken on bridging solution - logs every
+# Disable the feature. LoginAttempt seems to be broken on bridging solution - logs every
 # request instead of logins!
 # Follow up issue: https://github.com/silverstripe/cwp-core/issues/41
 # SilverStripe\Admin\LeftAndMain:
```

Comment:
```suggestion
```
Delete the whole todo comment and the related commented-out code please.

Reply:
Updated